### PR TITLE
DT-2509 Add ability to update Role Description

### PR DESCRIPTION
**Pull request by karenmillermoj**
There will be another PR to allow CRLF

### CODE REVIEW

File: backend/controllers/roleDescriptionAmendment.test.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      await changeRoleDescription.index(req, { render })
+      expect(render).toBeCalledWith('changeRoleDescription.njk', {
+        currentRoleDescription: 'role description',
+        title: 'Change role description',
+        roleUrl: '/manage-roles/role1',
+        errors: undefined,
+      })
+    })
+
+    it('should copy any flash errors over', async () => {
```
Comment (karenmillermoj) [inline backend/controllers/roleDescriptionAmendment.test.js:30]: I will be adding another test to pass thru  {} - next PR

File: backend/controllers/roleDescriptionAmendment.test.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        roleDescription: 'role description',
+      })
+
+      const render = jest.fn()
+      await changeRoleDescription.index(req, { render })
+      expect(render).toBeCalledWith('changeRoleDescription.njk', {
+        errors: { error: 'some error' },
+        currentRoleDescription: 'role description',
+        title: 'Change role description',
+        roleUrl: '/manage-roles/role1',
```
Comment (andymarke) [inline backend/controllers/roleDescriptionAmendment.test.js:42]: very minor, but would it make sense to be 'Change description for <role1>`  ??? rather then the description in title. I suppose we can see what it looks like for real data

**Review by andymarke — APPROVED** [review]
LGTM

File: backend/controllers/roleDescriptionAmendment.test.js (hunk 2 above)
Comment (karenmillermoj) [reply backend/controllers/roleDescriptionAmendment.test.js:42]: yes, I will update to be more meaningful in next pr

**Review by petergphillips — CHANGES_REQUESTED** [review]
LGTM

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**